### PR TITLE
FY2020Q3 content refresh: Identity M02

### PR DESCRIPTION
**Pull request by andrewconnell**
- updated screenshots where necessary
- updated steps & typos where necessary

### CODE REVIEW

File: Identity/02 Application Types/README.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ The lab for this module is available in multiple units within the associated Mic
...
+    > In this exercise, you'll learn how to create server-side web apps that allow users to sign in and grant the app permissions to act on the user's behalf. One the user has authenticated and granted the app consent to act on their behalf, the web application will use data returned from Microsoft Graph by using the OAuth 2.0 auth code grant flow.
 
 1. [Exercise - Daemon and non-interactive apps](https://docs.microsoft.com/learn/modules/identity-application-types/7-exercise-daemon-non-interactive-apps)
 
-    > In this exercise, you’ll learn how to create apps that obtain tokens without user interaction to either act on behalf of a user or perform tasks as the app’s identity.
+    > In this exercise, you'll learn how to create apps that obtain tokens without user interaction to either act on behalf of a user or perform tasks as the app's identity.
```
Comment (CelesteDG) [inline Identity/02 Application Types/README.md]: Add "to" before "perform"

File: Identity/02 Application Types/demos/01-spa/README.md [hunk 2]
```diff
@@ -12,5 +12,5 @@ This completed project is the result of the lab exercise **Single Page Applicati
 - Create an Azure AD application by following the instructions in the lab exercise associated with this demo. In this step, you are instructed to collect these data elements:
   - applicationId
   - directoryId
-- Update the properties in the **[index.html](./index.html.json)** with the values you collected in the last step.
+- Update the properties in the **[index.html](./index.html)** with the values you collected in the last step.
 - Build & run the application by following the instructions in the lab exercise associated with this demo.
```
Comment (CelesteDG) [inline Identity/02 Application Types/demos/01-spa/README.md]: Change "&" to "and"

File: Identity/02 Application Types/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Microsoft Identity Training Module - Application Types
 
-The Microsoft identity platform enables developers to build many different types of applications to satisfy diverse business requirements and different scenarios. By supporting multiple OAuth 2.0 standard authentication protocols, developers can create different types of applications that meet business needs including single page applications, web apps, mobile or native apps and services or daemon apps. In this module, you’ll learn how you can implement different authentication protocols in popular application types.
+The Microsoft identity platform enables developers to build many different types of applications to satisfy diverse business requirements and different scenarios. By supporting multiple OAuth 2.0 standard authentication protocols, developers can create different types of applications that meet business needs including single page applications, web apps, mobile or native apps and services or daemon apps. In this module, you'll learn how you can implement different authentication protocols in popular application types.
```
[the remark this answers is not in the document]
Comment (andrewconnell) [reply Identity/02 Application Types/README.md]: Fixed

File: Identity/02 Application Types/README.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ The lab for this module is available in multiple units within the associated Mic
 
 1. [Exercise - Single Page Applications](https://docs.microsoft.com/learn/modules/identity-application-types/3-exercise-single-page-applications)
 
-    > In this exercise, you’ll learn how to create, configure and use Azure AD applications that use data returned from Microsoft Graph in single page applications by leveraging the OAuth 2.0 implicit grant flow.
+    > In this exercise, you'll learn how to create, configure and use Azure AD applications that use data returned from Microsoft Graph in single page applications by leveraging the OAuth 2.0 implicit grant flow.
```
[the remark this answers is not in the document]
Comment (andrewconnell) [reply Identity/02 Application Types/README.md]: Fixed

File: Identity/02 Application Types/README.md (hunk 1 above)
Comment (andrewconnell) [reply Identity/02 Application Types/README.md]: Fixed

File: Identity/02 Application Types/README.md [hunk 5]
```diff
@@ -10,15 +10,15 @@ The lab for this module is available in multiple units within the associated Mic
...
 
 1. [Exercise - Web apps that sign in users & call APIs](https://docs.microsoft.com/learn/modules/identity-application-types/5-exercise-web-apps-call-apis)
 
-    > In this exercise, you’ll learn how to create server-side web apps that allow users to sign in and grant the app permissions to act on the user’s behalf. One the user has authenticated and granted the app consent to act on their behalf, the web application will use data returned from Microsoft Graph by using the OAuth 2.0 auth code grant flow.
+    > In this exercise, you'll learn how to create server-side web apps that allow users to sign in and grant the app permissions to act on the user's behalf. One the user has authenticated and granted the app consent to act on their behalf, the web application will use data returned from Microsoft Graph by using the OAuth 2.0 auth code grant flow.
```
[the remark this answers is not in the document]
Comment (andrewconnell) [reply Identity/02 Application Types/README.md]: fixed

File: Identity/02 Application Types/demos/01-spa/README.md (hunk 2 above)
Comment (andrewconnell) [reply Identity/02 Application Types/demos/01-spa/README.md]: fixed

**Pull request merged**